test/integration/tests/4453-detailed/files/LICENSE

```diff
@@ -0,0 +1,30 @@
+Copyright Axel Mannhardt (c) 2015
```

Comment:
This seems odd? I think we generally do not need license files for integration tests (I've seen that comment on previous PRs).

Reply:
It's not my own name. It's the name of the person who wrote the original bug report because I copy-pasted their code.

Reply:
Do you want me to remove this file then?

Reply:
Yes, I'd rather have `files` containonly the source files, no `.gitignore`, no `LICENSE`